snippets/base/admin/adminmodels.py

```diff
@@ -1019,7 +1019,7 @@ class JobAdmin(admin.ModelAdmin):
     ]
     fieldsets = [
         ('ID', {
-            'fields': ('id', 'job_status', 'snippet_name_linked', 'creator')
+            'fields': ('id', ('job_status', 'completed_on'), 'snippet_name_linked', 'creator')
```

Comment:
I missed this while reviewing: https://github.com/mozmeao/snippets-service/issues/1220